source/_integrations/ista_ecotrend.markdown

```diff
@@ -34,3 +34,37 @@ The **ista EcoTrend** integration exposes the last monthly readings as sensors.
 - **Water costs**: estimated costs in EUR
 
 Not all values may be available in your ista EcoTrend account. Cost estimation is an optional service that has to be booked by your property manager. Therefore, the cost sensors are deactivated by default.
+
+## Long-term statistics
+
+The **ista EcoTrend** integration allows you to import all your historical consumption readings from your ista EcoTrend account into long-term statistic entities. These entities can be displayed in your Home Assistant energy dashboard, providing a comprehensive view of your consumption data over time.
+
+### Identifying ista EcoTrend statistic entities
+
+The statistic entities imported via this integration have a `ista_ecotrend:` prefix. This prefix helps you identify these entities separately from other sensor statistics when setting up the long-term statistics in the energy dashboard.
+
+### Setting up long-term statistics in the energy dashboard
+
+To set up the **ista EcoTrend** long-term statistics in your Home Assistant energy dashboard, follow these steps:
+
+- **Access the energy configuration panel**
```

Comment:
the reason you didn't use numbered steps is that they are not really sequential, but different options?